CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
-## Changelog for [1.8.1] (2022-08-05)
-The following sections list the changes for 1.8.1
+## Changelog for [1.8.2] (2022-10-17)
+The following sections list the changes for 1.8.2
```

Comment:
What happened to 1.8.1?

Reply:
Sorry, I thought I copy-pasted and edited the text. Didn't realize I edited the original one. I'll fix it